> @Damans227 looks like there's a couple lint issues here. I'd recommend running `npm run type` locally to see the typescript errors, and also updating the pr title according to the PR Lint error here

@etr2460 Looks like there is a type mismatch: 

```
src/explore/components/controls/HiddenControl.tsx:29:31 - error TS2322: Type 'string | number | boolean | Function | any[] | Record<string, any>' is not assignable to type 'string | number | readonly string[] | undefined'.
  Type 'false' is not assignable to type 'string | number | readonly string[] | undefined'.

29   return <Input type="hidden" value={props.value} />;
                                 ~~~~~

  node_modules/@types/react/index.d.ts:2224:9
    2224         value?: string | ReadonlyArray<string> | number | undefined;
                 ~~~~~
    The expected type comes from property 'value' which is declared here on type 'IntrinsicAttributes & Pick<InputProps, "dir" | "form" | "slot" | "style" | "title" | "pattern" | "id" | "children" | "prefixCls" | "className" | ... 281 more ... | "bordered"> & ... 4 more ... & { ...; }'

```
If I change the type of value in the interface to `value: 'string | number | readonly string[] | undefined';` then there are no errors.

--- a/superset-frontend/src/explore/components/controls/HiddenControl.tsx
+++ b/superset-frontend/src/explore/components/controls/HiddenControl.tsx
@@ -21,7 +21,7 @@ import { Input } from 'src/common/components';
 
 interface HiddenControlsProps {
   onChange: () => void;
-  value: 'string | number | readonly string[] | undefined';
+  value: string | number | readonly string[] | undefined;
 }
 
 export default function HiddenControl(props: HiddenControlsProps) {

--- a/superset-frontend/src/explore/components/controls/HiddenControl.tsx
+++ b/superset-frontend/src/explore/components/controls/HiddenControl.tsx
@@ -21,7 +21,7 @@ import { Input } from 'src/common/components';
 
 interface HiddenControlsProps {
   onChange: () => void;
-  value: string | number | Record<string, any> | boolean | any[] | Function;
+  value: 'string | number | readonly string[] | undefined';
 }
 
 export default function HiddenControl(props: HiddenControlsProps) {

